src/pages/index.js

```diff
@@ -68,7 +116,7 @@ function renderCard(item) {
 profileEditButton.addEventListener("click", () => {
   const currentUser = userInfo.getUserInfo();
   nameInput.value = currentUser.name.trim();
```

Comment:
**Could be improved**

You can call `resetValidation` for the profile form as well to clean possible errors